src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php

```diff
@@ -283,37 +291,15 @@ protected function getAbsoluteFilePath($path) {
   }
 
   /**
-   * @when /^(?:|I )confirm the popup$/
-   */
-  public function confirmPopup() {
-    $this->getSession()->getDriver()->getWebDriverSession()->accept_alert();
-  }
-
-  /**
-   * @when /^(?:|I )cancel the popup$/
-   */
-  public function cancelPopup() {
-    $this->getSession()->getDriver()->getWebDriverSession()->dismiss_alert();
-  }
-
-  /**
-   * @When /^(?:|I )should see "([^"]*)" in popup$/
```

Comment:
Why are these being removed?

Reply:
they never worked in the first place. you can't run javascript to test for an alert when the alert blocks javascript from running.

Reply:
Thanks for splain'n